client/landing/stepper/utils/test/get-flow-from-url.ts

```diff
@@ -0,0 +1,19 @@
+import { getFlowFromURL } from '../get-flow-from-url';
```

Comment:
Thanks for making the improvements by adding unit tests!

Reply:
I just couldn't figure out why `getFlowFromURL` seemed to be not working properly! Great reminder that unit tests are also a great way to quickly iterate on code so that you don't have to refresh the browser and redo the flow over and over.